presto-tests/pom.xml

```diff
@@ -434,7 +434,7 @@
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
                             <includes>
-                                <include>**/tests/*.java</include>
+                                <include>**/test/*.java</include>
```

Comment:
I believe the intention here may have been to include everything not included in other profiles.  Unfortunately, it had a typo.